src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -532,8 +525,12 @@ public void updateSubscribers()
 
             //Buildings
             sendBuildingPackets(oldSubscribers, hasNewSubscribers);
+
+            //Fields
+            sendFieldPackets(oldSubscribers, hasNewSubscribers);
```

Comment:
I think this should need to be placed somewhere else as well. With an if (isDirty) check